src/components/FilterPanel/FilterPanelCheckboxWithOverflowMenu/FilterPanelCheckboxWithOverflowMenu.stories.js

```diff
@@ -28,13 +28,13 @@ storiesOf(
       id="checkbox-id"
     >
       <OverflowMenuItem
-        primaryFocus
         itemText="Option 1"
         onClick={action('OverflowMenuItem onClick')}
       />
       <OverflowMenuItem
-        itemText="Option 2"
+        itemText="Option 2 -- selected when menu opens"
         onClick={action('OverflowMenuItem onClick')}
+        data-floating-menu-primary-focus
```

Comment:
This isn't a prop that is documented, does it make sense to have it?

Reply:
It's the default value for `selectorPrimaryFocus` in `OverflowMenu`